src/cli/commands/init.js

```diff
@@ -89,12 +91,17 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       question: 'license',
       default: String(config.getOption('init-license')),
     },
+    {
+      key: 'private',
+      question: 'private',
+      default: '',
```

Comment:
If the default was `default: defaultFlag ? 'true' : 'false'` then it would eliminate a conditional below